spell/spell-crafter-goerli-workflow.md

```diff
@@ -110,7 +110,9 @@ PR: https://github.com/makerdao/spells-goerli/pull/<TODO>
 * [ ] Add new ChainLog Address in `addresses_goerli.sol` (e.g. Collateral Onboarding)
 * [ ] Run Tests `make test` or `make test match=<test_name>` to inspect debug traces
   * [ ] Ensure Good Coverage
-  * [ ] Ensure every test function is declared as public if enabled or private if disabled
+  * [ ] Ensure every test function is declared as `public`
+    * [ ] IF the test is enabled, it MUST NOT have the `skipped` modifier
+    * [ ] OTHERWISE, if the test is disabled, it MUST have the `skipped` modifier
```

Comment:
Nit: since there is only two checks here I would only use two checkboxes:
 1. Are all test functions public?
 2. Are all functions that needs to be run are actually not skipped?
```suggestion
  * [ ] Ensure every test function is declared as `public`
  * [ ] IF the test needs to run, it MUST NOT have the `skipped` modifier; OTHERWISE, it MUST have `skipped` modifier
```